syntax = "proto3";

import "Task.proto";
import "TaskStructures.proto";

message AddTaskResponse {
  bool result = 1;
}

message AddSubTaskResponse {
  bool result = 1;
}

message EditResponse {
  bool result = 1;
}

message DeleteResponse {
  bool result = 1;
}

message CompleteResponse {
  bool result = 1;
}

message ShowByLabelResponse {
  model.ManyTasksWithId tasks = 1;
}

message ShowParentsResponse {
  model.ManyTasksWithId tasks = 1;
}

message ShowTaskResponse {
  model.CompositeTask task = 1;
}

message ShowAllResponse {
  model.ManyCompositeTasks tasks = 1;
}

message SaveResponse {
  bool result = 1;
}

message LoadResponse {
  bool result = 1;
}
